src/Components/Endgame/Endgame.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useContext} from "react";
+import './Endgame.css'
+import { NavLink } from 'react-router-dom'
+import { setGameContext } from "../App/App";
+
+
+function Endgame({correctAnswers, category, difficulty}) {
+ const [name, setName] = useState("")
+ const setGameEnded = useContext(setGameContext)
+
+ const score = correctAnswers * 100
+
+ const submitScore = () => {
+
+   const scorePost = `
+     mutation createLeaderBoard {
+       createLeaderboard(input: {
+         name: "${name}"
+         score: ${score}
+         category: "${category}"
+         difficulty: "${difficulty}"
+       }) {
+         leaderboard {
+           name
+           score
+           difficulty
+           category
+         }
+         errors
+       }
+     }
+     `
+
+     fetch("https://listen-up-be.herokuapp.com/graphql", {
+       method: "POST",
+       headers: {
+         "Content-Type": "application/json"
+       },
+       body: JSON.stringify({
+         query: scorePost
+       })
+     })
+     .then(res => res.json())
+     .then(setName(""))
+     .then(setGameEnded(true))
+     .catch(err => console.log(err))
+   }
+
```

Comment:
Doesn't need to be done for this merge but we are using this fetch call in multiple files so we should look into creating an API Calls file and import this function so we can reduce lines of code.